tangelo/linq/qpu_connection/braket_connection.py

```diff
@@ -0,0 +1,183 @@
+"""
+    Wrappers around Braket API, to manage quantum experiments run with Braket from Tangelo.
+"""
+
+from enum import Enum
+
+from tangelo.linq.translator import translate_circuit
+from tangelo.linq.qpu_connection.qpu_connection import QpuConnection
+
+
+class SupportedBraketProviders(str, Enum):
+    """ List of the providers currently supported. Needs to be occasionally
+    updated in the future as new providers arise. """
+
+    # SV1, TN1 and DM1 devices (simulators)
+    AMAZON = "Amazon Braket"
+
+    # Hardware providers
+    IONQ = "IonQ"
+    RIGETTI = "Rigetti"
+    OXFORD = "Oxford"
+
+
+def refresh_available_braket_devices():
+    """Function to get the available gate-based devices on Braket.
+    Note: OFFLINE and RETIRED devices are filtered out.
+    Returns:
+        list of braket.aws.AwsDevice: Available gate-based Braket devices.
+    """
+    from braket.aws import AwsDevice
+
+    return AwsDevice.get_devices(
+        provider_names=[provider.value for provider in SupportedBraketProviders],
+        statuses=["ONLINE"])
+
+
+class BraketConnection(QpuConnection):
+    """ Wrapper around Amazon Braket API to facilitate quantum job management from Tangelo """
+
+    def __init__(self, s3_bucket=None, folder=None):
+        """
+        Initialize connection object. Requires Braket to be installed.
```

Comment:
We could provide a link (https://github.com/aws/amazon-braket-sdk-python), becasue the name of the package is not obvious (not "braket" but "amazon-braket-sdk-python").